pkgs/desktops/gnome-3/core/gnome-shell-extensions/default.nix

```diff
@@ -27,6 +27,8 @@ stdenv.mkDerivation rec {
   doCheck = true;
 
   nativeBuildInputs = [ meson ninja pkgconfig gettext glib ];
+  # 52 is required for tests
+  # https://gitlab.gnome.org/GNOME/gnome-shell-extensions/blob/3.30.1/meson.build#L25
   buildInputs = [ spidermonkey_52 ];
```

Comment:
Maybe we can move this to `checkInputs`.

Reply:
That would work. I was about to do it, but wondered if it would get picked up since it's detected in the main meson.build file, should've just done it :)